python/pyspark/ml/stat.py

```diff
@@ -100,7 +100,7 @@ def test(dataset, featuresCol, labelCol, flatten=False):
         return _java2py(sc, javaTestObj.test(*args))
 
 
-class Correlation(object):
```

Comment:
I don't feel strongly on this because this is virtually no-op up to my best knowledge. but i'm fine.

Reply:
For me, it is just one more thing, that requires additional thought, unless you're familiar with Python 2 and the history of the project.